### PR TITLE
Feature/set published dates on articles

### PR DESCRIPTION
**Pull request by jacqui**
Issue #56 

This moves the responsibility for setting the first and last published dates to the webiny API level, in the article model, which makes a lot more sense than doing it ad hoc in the google add-on. I'll have some code to delete in the add-on as a result :)

### CODE REVIEW

File: api/tinynews-models/src/plugins/graphql/resolver.ts [hunk 1]
```diff
@@ -11,6 +11,37 @@ import { FieldResolver } from "@webiny/commodo-graphql/types";
 
 type GetModelType = (context: Object) => any; // TODO: add commodo type when available
 
+// export const resolveCreate = (getModel: GetModelType): FieldResolver => async (
```
Comment (TylerFisher) [inline api/tinynews-models/src/plugins/graphql/resolver.ts]: Will we end up needing this?

File: api/tinynews-models/src/plugins/graphql/resolver.ts [hunk 2]
```diff
@@ -35,6 +66,9 @@ export const resolveCreateFrom = (getModel: GetModelType): FieldResolver => asyn
   // ensure the new version shares the same parent, slug and env (what's the env? think we're missing that one)
   newRevision.parent = baseRevision.parent;
   newRevision.slug = baseRevision.slug;
+
+  newRevision.firstPublishedOn = baseRevision.firstPublishedOn;
+  newRevision.lastPublishedOn = baseRevision.lastPublishedOn;
```
Comment (TylerFisher) [inline api/tinynews-models/src/plugins/graphql/resolver.ts:40]: This will actually get overridden on new publishes, correct?

**Review by TylerFisher — COMMENTED** [review]
Just a couple questions before I merge this.

File: api/tinynews-models/src/plugins/graphql/resolver.ts (hunk 1 above)
Comment (jacqui) [reply api/tinynews-models/src/plugins/graphql/resolver.ts]: Oh oops. I thought we would, but we don't as of right now. I'll remove it.

File: api/tinynews-models/src/plugins/graphql/resolver.ts (hunk 2 above)
Comment (jacqui) [reply api/tinynews-models/src/plugins/graphql/resolver.ts:40]: Yeah - I need to write the `publishArticle` mutation that will update the dates (or more likely, date, as in `lastPublishedOn` date, singular).

**Pull request merged**